package.json

```diff
@@ -53,6 +53,7 @@
   "dependencies": {
     "fake-xml-http-request": "^2.0.0",
     "route-recognizer": "^0.3.3",
+    "url-parse": "^1.4.7",
```

Comment:
To bundle `url-parse` in the output file, `url-parse` should lives in `devDependencies`.